implementations/rust/ockam/ockam_api/src/control_api/backend/relay.rs

```diff
@@ -37,23 +38,30 @@ impl HttpControlNodeApiBackend {
             },
             _ => {
                 warn!("Invalid method: {method}");
-                ControlApiHttpResponse::invalid_method(method, vec!["PUT", "GET", "DELETE"])
+                ControlApiHttpResponse::invalid_method(method, vec!["POST", "GET", "DELETE"])
             }
         }
     }
 }
 
 #[utoipa::path(
-    put,
+    post,
     operation_id = "create_relay",
     summary = "Create a new Relay",
-    path = "/{node}/relay",
-    tags = ["relay"],
+    description =
+"Create a new Relay, the main parameters are the destination node `to` and the `name`.
+The address inherits the name value, but it's possible to specify it to allow the creation
+of multiple relays with the same address to different nodes.
+The creation will be asynchronous and the initial status will be `down`.
+Note that a specific `ockam-relay` attribute is required to allow the relay creation in the
```

Comment:
"a specific `ockam-relay` attribute is required". We could explain that this attribute must be found in a credential provided by the caller of the API.